### PR TITLE
Ping 2688, PING-2791, Ping 2730 [MAIN] 

### PR DESCRIPTION
**Pull request by agit26milan**

### CODE REVIEW

**Comment by agit26milan** [comment]
mas @fajarism @arilukmanp @iwangsyah mintol di review ya

File: __tests__/screens/GroupSetting/GroupSetting.test.js [hunk 1]
```diff
@@ -1,184 +1,194 @@
...
-            goBack: jest.fn()
-        }
-        const route = {
-            params: {
-                focusChatName: 'Settings'
-            }
-        }
-
+    const {toJSON} = render(<GroupSetting navigation={navigation} route={route} />, {wrapper});
+    expect(toJSON).toMatchSnapshot();
```
Comment (fajarism) [inline __tests__/screens/GroupSetting/GroupSetting.test.js:174]: should be 
`expect(toJSON()).toMatchSnapshot();`

toJSON should be invoked --> toJSON() so it return the correct snapshot on .snap file

File: src/screens/ContactScreen/index.js [hunk 2]
```diff
@@ -123,7 +123,7 @@ const ContactScreen = ({navigation}) => {
       const members = followed;
       const channelName = usernames;
       let typeChannel = 0;
-      console.log(members, 'moon')
+      console.log(members, 'moon');
```
Comment (fajarism) [inline src/screens/ContactScreen/index.js:126]: remove console.log

**Review by fajarism — CHANGES_REQUESTED** [review]
Group setting test snapshot is wrong

**Review by fajarism — APPROVED** [review]

**Pull request merged**